sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/AbstractSormasToSormasInterface.java

```diff
@@ -226,10 +226,10 @@ public void sendRejectShareRequest(String uuid) throws SormasToSormasException {
 		SormasToSormasShareRequestDto shareRequest = shareRequestFacade.getShareRequestByUuid(uuid);
 
 		String organizationId = shareRequest.getOriginInfo().getOrganizationId();
-		sormasToSormasRestClient.post(organizationId, requestRejectEndpoint, uuid, null);
+//		sormasToSormasRestClient.post(organizationId, requestRejectEndpoint, uuid, null);
```

Comment:
Hmm, I think this should be uncommented otherwise the status won't be updated on the original instance